keyboards/viktus/smolka/smolka.c

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2021 jrfhoutx
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "smolka.h"
+
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    return encoder_update_user(index, clockwise);
+}
```

Comment:
```suggestion
```

Reply:
I left out the rest of the code accidentally, but it’s here because I’m trying to be consistent with previous requests, per 
https://github.com/qmk/qmk_firmware/pull/14069#discussion_r693602037
I will add the missing portion of code this evening, if that’s acceptable.

Reply:
code has been updated with the changes I mentioned above